### ISSUE TITLE
HTML Errors

### ISSUE THREAD

**Issue by tbillington** [prettier/prettier-atom#78]
Prettier-atom runs over html files when saving, causing errors on script tags.

Sample:
```html
<script async src="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.min.js"></script>
```

Error:
```
SyntaxError: Unexpected token (1:1)
> 1 | </script>
    |  ^
  2 |   </
```

**Comment by robwise** [comment]
Will fix, for the time being, I suggest taking `text.html.basic` out of your scopes

**Comment by robwise** [comment]
@transcranial Do you have any insight into what's going on here? It seems like if the script tags are on the same line, it errors?

**Comment by transcranial** [comment]
@robwise Hey sorry, didn't see this until now. Just made a pull request that should fix this. #123

**Comment by robwise** [comment]
Fixed in #123
